src/styles/pagination.m.css

```diff
@@ -0,0 +1,32 @@
+@import './shared/variables.css';
+
+.navigation {
+}
+
+.arrow {
```

Comment:
Since we pass our theme down several levels, this probably needs a more specific name than arrow